bin/otomi

```diff
@@ -14,7 +14,7 @@
 set -e
 
 # normalize $CI to be set to 'true' if set and not 'false'
-[ "$CI" != 'false' ] && [ "$CI" != '' ] && CI='true'
+{ [ "$CI" = 'true' ] || [ "$CI" != 'false' ]; } && CI='true'
```

Comment:
This change broke my test suite. 

I would like for us to start running the `bats` tests as the amount of tests are growing. 

**Whenever someone makes a change to our bash plumbing it should be immediately visible whether the change introduces a bug.**